<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableBlg</class>
 <widget class="QDialog" name="TableBlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1323</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#TableBlg{
	border-image: url(:/pictures/icon/frame_01.png);}</string>
  </property>
  <widget class="QPushButton" name="ok_button">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>470</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#ok_button{
border-image: url(:/pictures/icon/ok.png);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>251</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="choose_1">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;华文中宋&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="choose_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;华文中宋&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="choose_3">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;华文中宋&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="choose_4">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;华文中宋&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>251</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QCheckBox" name="choose_5">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;华文中宋&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="choose_6">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;华文中宋&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="choose_7">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;华文中宋&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="choose_8">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;华文中宋&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>70</y>
     <width>251</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QCheckBox" name="choose_9">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;华文中宋&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="choose_10">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;华文中宋&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="choose_11">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;华文中宋&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="choose_12">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;华文中宋&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>70</y>
     <width>251</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QCheckBox" name="choose_13">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;华文中宋&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="choose_14">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;华文中宋&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="choose_15">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;华文中宋&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="choose_16">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;华文中宋&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#label{
font: 75 14pt &quot;Arial&quot;;}</string>
   </property>
   <property name="text">
    <string>Please click to select:</string>
   </property>
  </widget>
  <widget class="QWidget" name="to_see_possession" native="true">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>480</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#to_see_possession{
border-image: url(:/pictures/icon/possession.png);}</string>
   </property>
  </widget>
  <widget class="QWidget" name="to_shop" native="true">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>480</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#to_shop{
border-image: url(:/pictures/icon/shop_icon.png);}</string>
   </property>
  </widget>
  <widget class="QWidget" name="Frog_pic" native="true">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>400</y>
     <width>151</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#Frog_pic{
border-image: url(:/pictures/frog_4.png);}</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
